### PR TITLE
Custom Card

### PR DESCRIPTION
**Pull request by kimberllyflorez**

### CODE REVIEW

File: lib/home/screens/home.dart [hunk 1]
```diff
@@ -3,6 +3,8 @@ import 'package:example_whit_github/profile/screens/profile.dart';
 import 'package:example_whit_github/theme/custom_colors.dart';
 import 'package:flutter/material.dart';
 
+import 'home_options.dart';
```
Comment (cesarurrego) [inline lib/home/screens/home.dart]: Why should use relative paths instead!

File: lib/home/screens/home.dart [hunk 2]
```diff
@@ -16,12 +18,10 @@ class _HomeState extends State<Home> {
   int _selectIndex = 0;
 
   static List<Widget> _children = <Widget>[
-    //Home(),
-    Text('sdfsdgdf'),
-    Text('sdfds'),
 
+    HomeOptions(),
+    Text('sdfds'),
```
Comment (curregoz) [inline lib/home/screens/home.dart:23]: Profile page should be second item

File: lib/widgets/text_fields/custom_card_field.dart [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+            height: 150,
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+              children: [
+                Text(TextTitle),
+                Text(TextDuration),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: [
+                    Icon(Icons.star_border_sharp),
```
Comment (cesarurrego) [inline lib/widgets/text_fields/custom_card_field.dart]: What happen with other 4 stars?

File: lib/home/screens/home_options.dart [hunk 4]
```diff
@@ -1,14 +1,30 @@
...
   @override
   Widget build(BuildContext context) {
-    return Container();
+    return Scaffold(
+      body: ListView(
+        padding: EdgeInsets.symmetric(horizontal: 10,vertical: 10),
+        children: [
+          CustomCard(
+              ImagenPageCard:
+              Image(image: NetworkImage("https://w7.pngwing.com/pngs/369"
```
Comment (cesarurrego) [inline lib/home/screens/home_options.dart]: what happen if we put a diferent size of image?

File: lib/widgets/text_fields/custom_card_field.dart [hunk 5]
```diff
@@ -0,0 +1,84 @@
+import 'package:example_whit_github/theme/custom_colors.dart';
+import 'package:flutter/material.dart';
+
+class CustomCard extends StatelessWidget {
+  final String TextTitle;
+  final String TextDuration;
+  final String? TextTiquet;
```
Comment (cesarurrego) [inline lib/widgets/text_fields/custom_card_field.dart]: parameters must be with cameCase

**Review by cesarurrego — CHANGES_REQUESTED** [review]

File: lib/home/screens/home_options.dart (hunk 4 above)
Comment (kimberllyflorez) [reply lib/home/screens/home_options.dart]: i don't now

**Pull request merged**